src/components/standalone/firewall/rules/CreateOrEditFirewallRuleDrawer.vue

```diff
@@ -720,7 +720,8 @@ async function saveRule() {
       return tag.id
     }),
     log: isLoggingEnabled.value,
-    system_rule: props.currentRule?.system_rule || false
+    system_rule: props.currentRule?.system_rule || false,
+    active_zone: props.currentRule?.active_zone || false
```

Comment:
If I'm not mistaken `active_zone` should not be passed, since it is computed by the backend

Reply:
type script makes me some noise, crying I have a missing property in vscode

Reply:
I recall also the build failed, the previous name of the property was true_zone, I renamed it after active_zone

https://github.com/NethServer/nethsecurity-ui/actions/runs/12596455890/job/35107582027